Update README.md (#3)
EpionHealth/interview: README.md

Thanks for being interested in coming to work here at Epion Health. Our team values building, reviewing, and pairing on well-crafted applications.  This repo was built to test how you build applications and how you would fit in working with the team at Epion.  It's a simplified version of our main Check-in application, and will give you some insight into our product development process.  If you'd like to learn more about our Check-in product please visit our [website](http://www.epionhealth.com/checkin-how-it-works-2/)

> Note: If you don't have a preference we will use Slack to share screen, so please make sure you have the desktop version installed beforehand.

Reviewer: ```suggestion
We will use Slack to share your screen unless that wouldn't work for you. Please make sure you have the desktop app installed beforehand.
```
